SH4RE/SH4RE/CreateListing/CreateListingView.swift

```diff
@@ -178,7 +176,7 @@ struct CreateListingView: View {
                 .padding(.bottom)
             
             // availability
-            DropdownMenu(label: "Availability", options: availabilityList, selection: $availabilitySelection)
+            DropdownMenu(label: "Availability for the next 3 months", options: availabilityList, selection: $availabilitySelection)
```

Comment:
I think having "for the next 3 months" is a liiiiittle strange but I also get why you did it, and don't see too much of a way around it. Ideally, we would have had a system that made it available every day forever then only allow bookings to be made in the next 3 months for example, but this is okay for demo purposes I guess.
Also, plz make it capitalized. I think Cost Per Day should have also been capitalized. So "Availability For The Next 3 Months" or "Availability for the Next 3 Months" or some variation of that

Reply:
will change to "Availability for the Next 3 Months"